DQM/TrackingMonitor/src/TrackEfficiencyMonitor.cc

```diff
@@ -282,8 +303,22 @@ void TrackEfficiencyMonitor::analyze(const edm::Event& iEvent, const edm::EventS
   iSetup.get<IdealMagneticFieldRecord>().get(bField); 
   iSetup.get<TrackerRecoGeometryRecord>().get(theTracker);
   theNavigation = new DirectTrackerNavigation(theTracker);
-  
-  
+
+  edm::Handle<edm::View<reco::Muon> > muons; 
+  iEvent.getByToken(muonToken_,muons);
```

Comment:
@ocolegro please add a check on the validity of the muon collection here, this part may cause issues in the Online clients if the muons are not reconstructed

Reply:
I agree on the comment by @fioriNTU A prevention is needed please